cpp/oneapi/dal/algo/svm/BUILD

```diff
@@ -16,3 +20,14 @@ dal_module(
         "@onedal//cpp/daal/src/algorithms/svm:kernel",
     ]
 )
+
+dal_test_suite(
+    name = "cpu_tests",
+    srcs = [
+        # Diabled
+        # "backend/cpu/train_kernel_test.cpp",
```

Comment:
Need to uncomment? 
Especially since we can disable tests straight into their sources: https://stackoverflow.com/questions/7208070/googletest-how-to-skip-a-test

Reply:
Ok